.eshintrc

```diff
@@ -0,0 +1,165 @@
+{
+  "env": {
+    "jquery": true,
+    "browser": true,
+    "node": true
+  },
+  "globals": {
+    "Model": false
+  },
+  "ecmaFeatures": {
+
+  },
+  "rules": {
+    "no-comma-dangle": 2,
+    "no-cond-assign": [2, "except-parens"],
+    "no-console": 1,
```

Comment:
The values mean:
* `0` is disabled
* `1` is a warning
* `2` is an error

There are also different modes for eslint (browser, node, …). At least they provide the default keywords and params for the mode. Also some rules are special then (see the node rules section). Not sure what will happen to this rule in node. But I would guess it’ll throw a warning.